kalite/coachreports/templates/coachreports/tabular_view.html

```diff
@@ -86,7 +86,7 @@
             <select id="report_type">
                 <option {% if not request_report_type %}selected{% endif %}>----</option>
                 {% for report_type in report_types %}
-                    <option value="{% trans report_type %}" {% if report_type == request_report_type %}selected{% endif %}>{{ report_type }}</option>
+                    <option value="{{ report_type.value }}" {% if report_type.value == request_report_type %}selected{% endif %}>{{ report_type.name }}</option>
```

Comment:
This allows coach reports to work in all languages - previously would break as the value was getting translated and then checked in English.